### DISCUSSION TITLE
Halloween Event - Victoria Sisters NFT + LORE

### DISCUSSION THREAD

**Discussion by NanndoSnowzera** [sunflower-land/sunflower-land#1350] [Art]
**Topic update:**
**Guys, we are updating the topic for a Halloween Event proposal:**
**New Theme:** Pumpkins and map will be changed to halloween theme.
**New Sounds:** Some sounds were made for the map, sounds for effects and also for the Halloween NFTs, all credits from the creators fixed in the file available for download.
**How to Participate:** Pumpkins will have a chance to drop Jack O'Lantern during harvest, these (Jack O'Lantern are used to buy NFT's at the Halloween shop.
So plant pumpkins while you can **(Valid from October 31st to November 7th)....**
**Jack O'Lantern:** Has a 15% chance to drop from any pumpkin you harvest during the event.
When you acquire 1 Jack O'Lantern during harvest, there will be an animation over the pumpkin you harvested, after it disappears.
Enjoy the tricks and treats of the sunflower lands and have fun!

**NFT Sisters of Victoria:** 
![Sisters of Victoria](https://user-images.githubusercontent.com/105903391/197631400-49e91f2d-3de6-4ccd-82ae-e0c701498ec6.gif)

**Small Version:** 
![Small Version Sisters of Victoria](https://user-images.githubusercontent.com/105903391/197631423-897db2d0-3ce8-4da0-86c9-7892fedc491d.gif)

**"BOOST" Sisters of Victoria:** this is our first halloween so i thought we could have something epic.
for cats, I thought of a balance of a little more than a rabbit and a little less than a golden cauliflower.
what would be a 60% pumpkin harvest bonus = 1.6x...

**Cauldron V1:**
![IDLE and ON CLICK](https://user-images.githubusercontent.com/105903391/197676318-b45e5226-8558-43b2-914a-f2007456d521.gif)

**Cauldron Small Version:** 
![Cauldron](https://user-images.githubusercontent.com/105903391/197631529-27d897c4-1d30-4fa4-9e4b-9534aa5d95de.gif)

**Cauldron on click:**  
![cauldronv2-onclick](https://user-images.githubusercontent.com/105903391/197631602-412f31d0-1be2-477a-bee9-3cb93a8f1f96.gif)

**"Decorative" Mysterious Magic Cauldron:** it is something mysterious and decorative. a spell is sealed in that cauldron. it's better to have that magic on your side...  
![eye2](https://user-images.githubusercontent.com/105903391/197682058-adf1e9e9-52fe-4b49-a007-e8ae73a940a5.png)

**SUPPLY:** Time limited (Mint only available during event week for everyone and never comes back after event ends).
**Price of NFTs:** (Possibility to be changed).
**Cats price =** 170 Jack O'Lantern
**Cauldron Price =** 80 Jack O'Lantern
**Chance of dropping Jack O'Lantern =** 15%

**(PURE) animation of when you drop jack o'lantern from normal pumpkin:**
![Animation Pure Example](https://user-images.githubusercontent.com/105903391/197631253-9f7a58aa-9676-4405-a275-59cff550ab43.gif)

**(IN-GAME) animation of when you drop jack o'lantern from normal pumpkin:**
![Animation Effect when Harvest Jack O'lantern in Game](https://user-images.githubusercontent.com/105903391/197631206-68d6967c-fe56-485c-91a6-a03f69c47169.gif)

**30% discount For those who have the NFT Victoria the cat.**
Menu informing when an owner of Victoria the cat enters the game during the Halloween event.
**Victoria the cat:** heyyy heyy... It's Halloween!!! I'm Victoria the cat, I came to help you bring my sisters back to me.
Accept my help and take this discount to bring them back.

**Airdrop with 15un Jack O'lanterns For those who have the NFTs Sunflower Tombstone, War Tombstone and War Skull.**
You are the commander of death. we offer you a airdrop with 15 Jack O'Lantern to help you on this journey of getting the halloween NFTs, enjoy.
**Proposal: ALERT!!! (this idea may change, or not enter the event).** 

**Can I withdraw Halloween items?**
Jack O'Lantern exchange available after (4) four days of event.
NFTs negotiable after the event ends.

**Start date:** October 31st
**End of event:** November 7th

**Can I withdraw Halloween items?**
Jack O'Lantern exchange available after four days of event.
NFTs negotiable after the event ends.

**Start date:** October 31st
**End of event:** November 7th

**Link: Download link of the complete file with information**
https://www.mediafire.com/file/vs4a1obmrawvrud/Halloween_News_SFL.7z/file

**Credits for ideas, creations, map, NFTs, effects, helps, it's all in this file. all credits and mentions. thank you all for the effort and contribution.**
**Link Download with Credits:**
https://www.mediafire.com/file/xj422j9p553rw9d/All_Credits_Halloween.7z/file

~ WeAreSFLweAreLegion =) 

                                         HALLOWEEN LORE VICTORIA SISTERS:

![LORE 1](https://user-images.githubusercontent.com/105903391/185741889-6f7717b6-f259-4907-9ef7-62c8b4ab346f.png)
![LORE 2](https://user-images.githubusercontent.com/105903391/185742434-3e7815ee-f398-46eb-8464-c47f49c28fd1.png)

_Text content for those who want to translate:_

**LORE:** One day on a farm on the outskirts of Sunflower lands, 4 cats were born, one was named victoria and the other 3 had no such privilege of being named by their owners, the owners of the farm where the black cats were born, believed that they would bring bad luck in the harvest. so one of their distant acquaintances who came to visit them decided to adopt them and name them Talya, Lucy and Valkiria. Victoria's sisters' new owners also owned a farm.
one day in an October harvest, his pumpkins had never had a harvest as big as this one. your pumpkins were born 2x bigger than normal. and when they went to see the pumpkins, the little sisters of victory had fun playing on top of a huge pumpkin. it was the biggest pumpkin ever picked in those surroundings, so they were sure the victory sisters brought good luck.
So they decided to take victoria sisters to visit her and they also took with them the biggest pumpkin they picked and they also took their great luck with them to the sunflower lands.

**Comment by DaewiTrain** [comment]
I need this Halloween event, I need a kitten, I need to be lucky.

**Comment by VadimasVP** [comment]
I like the idea, because the game needs such events.

I want to say a little about the NFT itself. I would add some shades of yellow to the eyes and mouth of the pumpkin. About the cats. Adam once said that we should avoid pure black colors, because it's very conspicuous.

I have an example, I did it quite a long time ago. Maybe you will find something useful in this example.
![Cat8](https://user-images.githubusercontent.com/102822587/185756090-0bff4b63-4523-42a1-ab5c-1c026b18d23b.png)

**Reply by Telknub** [reply]
The idea for them to be fully black is to be more misteryous and scary, so they being conspicuous actually has a purpose since it fits the Halloween theme better. About the yellow shades on the pumpkin, I tried but it was blending with the pumpkins colors, but I can look at it later.

**Comment by denis3333** [comment]
I loved the idea and the nft, I really want it on my farm

**Comment by srfantasmax** [comment]
We need that lore. It's great

**Comment by tiffanydys** [comment]
Love this, would definitely be good to release around october/november to help with the spooky vibe.

**Comment by ghost** [comment]
Very nice!

**Comment by MrSnorch** [comment]
Great suggestion! 🚀

**Comment by Basosd** [comment]
It looks great! Great idea 🚀

**Comment by Crypto-Joan** [comment]
Interesting suggestion....let's do it

**Comment by WarDog23** [comment]
It's so cool, I need to "pump" my pumpkins now! LOL

**Comment by Whaitte** [comment]
Nice

**Comment by NanndoSnowzera** [comment]
**updated topic.**

**Comment by SuperSynapse** [comment]
This is fun, I like the idea of another side story with 3 black cats for Halloween :D

**Comment by JcEii** [comment]
can't wait to see this <3

**Comment by Cicciolin0** [comment]
this is a must have NFT for the halloween event!!!

**Comment by SakataNFT** [comment]
great , miau

**Comment by adamhannigan** [comment]
@NanndoSnowzera We love the lore around the cats and are looking to incorporate this for this years Halloween event.

We will need to adjust the requirements a bit so it is more accessible than just 500 players and will introduce a new 'Jack O Latern' Pumpkin type to prevent people just buying Pumpkins to get this item. Ideally we want the players to grind hard for the Bumpkins.

We will need to do a few iterations on the provided art as there are currently a couple problems

1. The width is too large (36px). Now that players can place items in Sunflower Isles, we need the max size of decorations/rare items to be 30-32px wide. I've included a thinner example where the cats fit on 2 squares wide.
2. The black cats look great against a white background but look a bit off when used in the game. Everything else is so bright and cheerful that it does not fit in with the aesthetic.

See attached for examples. Do you have ideas around the above ^ 

We will also need art for a Jack O Lantern shape 😄 

![example_island-export-export](https://user-images.githubusercontent.com/11745561/196334967-986d100d-9e4d-4ca7-b5ea-dd6bf8a56426.png)

**Reply by NanndoSnowzera** [reply]
ok, we will work on an idea to improve the art, we welcome feedback, please give us your opinion on the color issue, do you think a grayish black would look better in the color of cats?

**Reply by adamhannigan** [reply]
@NanndoSnowzera It is all subjective so I can't give a clear colour or answer unfortunately.

A dark gray may still look a bit of in Sunflower Land as well 🤔 Would the cat still be spooky if it was a light gray?

**Reply by Myth727** [reply]
Personally I love both and think they blend in well...
![Screenshot_20221018-185710_Firefox](https://user-images.githubusercontent.com/96928557/196560628-6ef01344-4be4-485f-bebc-c6c146870559.jpg)

**Reply by Telknub** [reply]
Made a few changes on the design here:

Old:
![195203502-9e187e15-aff1-4b83-8bc9-a988543c2ed7-export](https://user-images.githubusercontent.com/38796969/196811021-7b36041e-82b0-4107-8428-a33aff572697.png)

New:
![Halloween](https://user-images.githubusercontent.com/38796969/196811105-e5284aa7-feb6-4c2e-8ce4-8bb05ea0811c.png)

Comparison: (new one is the down left one)
![196334967-986d100d-9e4d-4ca7-b5ea-dd6bf8a56426](https://user-images.githubusercontent.com/38796969/196811337-430446ad-ec37-4193-bdd9-f03669c5a296.png)

What do you think? Is it better? Any other changes? 

I have every asset (the cats and the Jack o'lantern) separated if you need any of them. If you think its good enough I will make the animations on the new one and send everything on the right size as well.
